scripts/elasticsearch_upgrade.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+set -o pipefail
+
+# Check upgrade flag
+if ! [[ "$UPGRADE_ELASTICSEARCH" == "YES" ]]; then
```

Comment:
This should not be here. The only entry point for this script is through the `post-leap.sh` script. If this script is called it should just run.

Reply:
fixed.